exporter/syslogexporter/sender.go

```diff
@@ -186,17 +186,24 @@ func populateDefaults(msg map[string]any, msgProperties []string) {
 }
 
 func (s *sender) formatRFC3164(msg map[string]any, timestamp time.Time) string {
-	msgProperties := []string{priority, hostname, message}
+	msgProperties := []string{priority, hostname, message, app}
 	populateDefaults(msg, msgProperties)
 	timestampString := timestamp.Format("Jan 02 15:04:05")
-	return fmt.Sprintf("<%d>%s %s%s", msg[priority], timestampString, msg[hostname], formatMessagePart(msg[message]))
+	a := ""
```

Comment:
What is `a`? The variable name should reflect what it's for.

Reply:
Discussible: https://google.github.io/styleguide/go/decisions#variable-names

However I personally prefer longer names, so happy to change it to something more descriptive